homeassistant/components/epson/media_player.py

```diff
@@ -123,7 +123,11 @@ async def set_unique_id(self) -> bool:
 
     async def async_update(self) -> None:
         """Update state of device."""
-        power_state = await self._projector.get_power()
+        try:
+            power_state = await self._projector.get_power()
+        except ProjectorUnavailableError as ex:
+            self._attr_available = False
+            return
         _LOGGER.debug("Projector status: %s", power_state)
         if not power_state or power_state == EPSON_STATE_UNAVAILABLE:
```

Comment:
I think this might be not needed anymore as it can't occur if you catch exception `power_state == EPSON_STATE_UNAVAILABLE`

Reply:
Yes, you are right. This should now only handle `power_state` being `False`

```suggestion
        if not power_state:
```

Moreover, line
```Python
        _LOGGER.debug("Projector status: %s", power_state)
```
should be moved after this if statement, not before.